clients/docs/includes/configs/cloud/java-sr.config

```diff
@@ -1,10 +1,13 @@
-# Kafka
+# Kafka client (producer/consumer/admin)
 bootstrap.servers={{ BROKER_ENDPOINT }}
 security.protocol=SASL_SSL
 sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="{{ CLUSTER_API_KEY }}" password="{{ CLUSTER_API_SECRET }}";
```

Comment:
We will need to test this PR to validate removal of `ssl.endpoint.identification.algorithm`, just to double-check.

There are also many references in docs with this configuration parameter.  Do those need to be modified as well?

```
cp-component/connect-cloud-config.rst:        ssl.endpoint.identification.algorithm=https
cp-component/connect-cloud-config.rst:        consumer.ssl.endpoint.identification.algorithm=https
cp-component/connect-cloud-config.rst:        producer.ssl.endpoint.identification.algorithm=https
cp-component/connect-cloud-config.rst:         database.history.consumer.ssl.endpoint.identification.algorithm=https
cp-component/connect-cloud-config.rst:         database.history.producer.ssl.endpoint.identification.algorithm=https
cp-component/connect-cloud-config.rst:        ssl.endpoint.identification.algorithm=https
cp-component/connect-cloud-config.rst:        consumer.ssl.endpoint.identification.algorithm=https
cp-component/connect-cloud-config.rst:        producer.ssl.endpoint.identification.algorithm=https
cp-component/connect-cloud-config.rst:            "database.history.consumer.ssl.endpoint.identification.algorithm": "https",
cp-component/connect-cloud-config.rst:            "database.history.producer.ssl.endpoint.identification.algorithm": "https",
cp-component/c3-cloud-config.rst:        confluent.controlcenter.streams.ssl.endpoint.identification.algorithm=https
cp-component/c3-cloud-config.rst:        confluent.monitoring.interceptor.ssl.endpoint.identification.algorithm=https
migrate-topics-on-cloud-clusters.rst:    ssl.endpoint.identification.algorithm=https
migrate-topics-on-cloud-clusters.rst:    ssl.endpoint.identification.algorithm=https
migrate-topics-on-cloud-clusters.rst:        "src.kafka.ssl.endpoint.identification.algorithm":"https",
migrate-topics-on-cloud-clusters.rst:        "dest.kafka.ssl.endpoint.identification.algorithm":"https",
migrate-topics-on-cloud-clusters.rst:  ssl.endpoint.identification.algorithm=https
migrate-topics-on-cloud-clusters.rst:  producer.ssl.endpoint.identification.algorithm=https
migrate-topics-on-cloud-clusters.rst:   ssl.endpoint.identification.algorithm=https
migrate-topics-on-cloud-clusters.rst:     ssl.endpoint.identification.algorithm=https
```

Reply:
Possibly, but I'd prefer the teams owning those areas to verify. Note that the setting is redundant if you're using a client released within the last 2+ years. It doesn't harm. Also, for CCloud, it's always been optional.

Reply:
> Note that the setting is redundant if you're using a client released within the last 2+ years. It doesn't harm. 

@ijuma if this has been unnecessary for the last 2+ years, that is good enough for me.  (In comparison, if it were a new option introduced in 2.6).

Note: we will still want to validate so please allow us some time to do so.

Reply:
Totally agreed. That's why I added `client.dns.lookup=use_all_dns_ips` even though it's the default in 2.6.

Reply:
@ijuma apologies if I'm just missing it, but I don't see `client.dns.lookup=use_all_dns_ips` in the PR yet?

Reply:
Good point, I somehow lost this change when I moved the PR from the other repo to this one. Fixed.